include/irr/asset/ICPUMeshBuffer.h

```diff
@@ -84,6 +84,14 @@ class ICPUMeshBuffer : public IMeshBuffer<ICPUBuffer, ICPUDescriptorSet, ICPURen
     {
         return &m_indexBufferBinding;
     }
+	inline void setIndexBufferBinding(asset::ICPUBuffer& buffer)
+	{
+		m_indexBufferBinding.buffer = irr::core::make_smart_refctd_ptr<asset::ICPUBuffer>(buffer);
+	}
+	inline void setIndexBufferBinding(irr::core::smart_refctd_ptr<asset::ICPUBuffer>& buffer)
```

Comment:
this function doesnt let setting offset in binding, it'd be better for it to take SBufferBinding&&

Reply:
resolved

Reply:
see https://github.com/Crisspl/IrrlichtBAW/pull/13#discussion_r334249686

Reply:
remove this function